src/raftstore/store/debug.rs

```diff
@@ -63,6 +85,23 @@ impl Debugger {
         Debugger { engines }
     }
 
+    /// Get all region id in the specified CF, only used for KV rocksdb.
+    pub fn get_all_regions(&self, cf: &str) -> Result<Vec<u64>> {
```

Comment:
In what case `cf` is not `CF_RAFT`?

Reply:
For now there is not because we only have `get_region_info` using it. However, I think maybe we will need to get regions from other CFs in future. So keep the CF as an argument is better?